osu.Game.Tests/NonVisual/Filtering/FilterQueryParserTest.cs

```diff
@@ -256,7 +254,7 @@ public void TestCustomKeywordIsParsed()
 
         private class CustomFilterCriteria : IRulesetFilterCriteria
         {
-            public string CustomValue { get; set; }
+            public string? CustomValue { get; set; }
```

Comment:
Curious should it be set as nullable?

Reply:
Either or, but don't have to in this case because `null` isn't necessary in this usage.